dev/cognitiveservices/data-plane/Language/questionanswering-generate.json

```diff
@@ -284,10 +284,19 @@
           "$ref": "common.json#/definitions/JobState"
         },
         {
-          "$ref": "#/definitions/GenerationResult"
+          "$ref": "#/definitions/GenerationJobResult"
         }
       ]
     },
+    "GenerationJobResult": {
+      "description": "Generation Job result represents the job result with extracted documents and question answers generated.",
+      "type": "object",
+      "properties": {
+        "result": {          
+          "$ref": "#/definitions/GenerationResult"
```

Comment:
GenerationResult.documents is referencing a type that doesn't seem to exist in this file. Should `Document` -> `GenerationDocument` on L308?

Reply:
Correct @kristapratico . Thanks.